clients/filesystem-hadoop3/src/test/java/com/datastrato/gravitino/filesystem/hadoop3/TestOauth2Client.java

```diff
@@ -158,6 +161,33 @@ public void testFileSystemAuthUnauthorized() {
     // should throw UnauthorizedException
     assertThrows(
         UnauthorizedException.class, () -> managedFilesetPath.getFileSystem(configuration));
+
+    // test wrong client secret
```

Comment:
Expired token or wrong token. I don't want to verify the OAuth server. I would like to test the Gravitino server.

Reply:
The Mock Gravitino Server in the current unit test is a relatively simple REST service without a `ServerAuthenticator` environment, I may be more inclined to put this test into the integration test.

Reply:
OK.